### PR TITLE
fix(EditableText): stop push content with elipsis

### PR DESCRIPTION
**Pull request by romainseb**
**What is the problem this PR is trying to solve?**
The current editable text push the content when we over it to show the pencil icon

**What is the chosen solution to this problem?**
We have to :
- Ensure we can focus the pencil with the keyboard
- Keep some space in the elipsis to place the icon
- The button has not to be sr-only as it's not a content for the screen reader only

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/stories/Drawer.js [hunk 1]
```diff
@@ -217,14 +217,39 @@ const editableDrawers = [
...
 ];
 
+const longEditableDrawers = [
+	<Drawer
+		stacked
+		title="Im stacked drawer 1"
+		footerActions={Object.assign({}, basicProps, { selected: 0 })}
+	>
+		<h1>Hello drawer 1</h1>
+		<p>{"You should not being able to read this because I'm first"}</p>
```
Comment (jare-talend) [inline packages/components/stories/Drawer.js]:
```suggestion
		<p>You should not being able to read this because I'm first</p>
```

**Comment by jare-talend** [comment]
cross
The button delete move slightly

File: packages/components/src/Drawer/Drawer.scss [hunk 2]
```diff
@@ -69,19 +69,14 @@ $tc-drawer-z-index: $drawer-z-index !default;
...
-
-	&:hover {
-		:global(.tc-editable-text-pencil) {
-			opacity: 1;
-			&:hover {
-				color: $scooter;
-			}
+	:global {
+		.tc-editable-text {
+			width: 90%;
```
Comment (jare-talend) [inline packages/components/src/Drawer/Drawer.scss]:
```suggestion
			min-width: 0;
```

Comment (jare-talend) [reply packages/components/src/Drawer/Drawer.scss]: You use flexibility. The button close don't move with that.

**Comment by romainseb** [comment]
> I don't understand why you remove the sr-only. This class is a way to mask accessibility an element. For every one, the element gone away but for the people using a screen reader, it can see it.

It's the opposite, sr-only class is placed on elements we don't want to show to non screen reader users but when we need to give some information to *screen reader only*.
In this case, we want the button for the users using a screen reader or not 😄

File: packages/components/src/Drawer/Drawer.scss [hunk 3]
```diff
@@ -69,19 +69,14 @@ $tc-drawer-z-index: $drawer-z-index !default;
...
-				color: $scooter;
-			}
+	:global {
+		.tc-editable-text {
+			min-width: 0;
 		}
 	}
+	:global(.tc-editable-text-pencil) {
+		color: $white;
+	}
```
Comment (ncomont) [inline packages/components/src/Drawer/Drawer.scss]:
```
	:global {
 		.tc-editable-text {
 			min-width: 0;
 		}
 	        .tc-editable-text-pencil) {
 		        color: $white;
 	}
```

**Pull request merged**